docs/getting-started/installation.mdx

```diff
@@ -23,131 +23,151 @@ Payload requires the following software:
-})
+```
+https://github.com/payloadcms/payload/tree/beta/templates/blank-3.0/src/app/(payload)
 ```
 
-This server doesn't do anything just yet. But, after you have this in place, we can initialize Payload via its asynchronous `init()` method, which accepts a small set of arguments to tell it how to operate.
-
-To initialize Payload, update your `server.ts` file to reflect the following code:
-
-```ts
-import express from 'express'
-import payload from 'payload'
-
-require('dotenv').config()
-const app = express()
-
-const start = async () => {
-  await payload.init({
-    secret: process.env.PAYLOAD_SECRET,
-    express: app,
-  })
-
-  app.listen(3000, async () => {
-    console.log('Express is now listening for incoming connections on port 3000.')
-  })
-}
+Notice how the Payload files are all kept within the `(payload)` [Route Group](https://nextjs.org/docs/app/building-your-application/routing/route-groups)? That's so that the Payload admin UI can have its own separate [Root Layout](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#root-layout-required) which will allow it to not interfere whatsoever with an existing Next.js app if you have one. 
 
-start()
-```
+<Banner type="warning">
+  You may need to copy all of your existing frontend files, including your existing root layout, into its own newly created route group.
+</Banner>
 
-A quick reminder: in this configuration, we're making use of environmental variables, `process.env.PAYLOAD_SECRET`. Often, it's smart to store these values in an `.env` file at the root of your directory and set different values for each of your environments (local, stage, prod, etc). The `dotenv` package is very handy and works well alongside of Payload. A typical `.env` file will look like this:
+Once you have the required Payload files in place in your `/app` folder, you should have something like this:
 
 ```
-DATABASE_URI=mongodb://127.0.0.1/your-payload-app
-PAYLOAD_SECRET=your-payload-secret
+/app
+  -- (payload) // This is the Payload route group
+     -- layout.tsx // Payload's root layout
+     -- admin // The Payload admin UI
+     -- api // Payload's REST / GraphQL handlers
+     -- custom.scss // Define custom SCSS for the Payload admin here
+  -- (my-app) // this is where your existing app goes
+     -- layout.tsx // move your existing root layout here
+     -- page.tsx // your existing home page
+     -- ...etc // whatever else you have
 ```
 
-Here is a list of all properties available to pass through `payload.init`:
-
-#### secret
-
-**Required**. This is a secure string that will be used to authenticate with Payload. It can be random but should be at least 14 characters and be very difficult to guess.
+You can name the `(my-app)` folder anything you want. The name does not matter and will just be used to clarify your directory structure for yourself. Common names might be `(frontend)`, (`app`), or similar.
```

Comment:
(`app`) should be `(app)`

Reply:
Could link out to `../admin/overview` from here. That page is a continuation of this information.

Reply:
done and done